Libplanet.Headless.Tests/Hosting/LibplanetNodeServiceTest.cs

```diff
@@ -70,7 +70,7 @@ public void PropertiesMustContainGenesisBlockOrPath()
 
         private class BlockPolicy : IBlockPolicy<DummyAction>
         {
-            public IComparer<IBlockExcerpt> CanonicalChainComparer { get; } = new TotalDifficultyComparer();
+            public IComparer<BlockPerception> CanonicalChainComparer { get; } = new TotalDifficultyComparer(TimeSpan.FromSeconds(3));
```

Comment:
I guess you should mean 30 seconds?

Reply:
I don't have ideas about proper value so I copied from [here](https://github.com/planetarium/libplanet/blob/2cebb4906ad9b1ce58df81f85d4e9f90293f219d/Libplanet.Tests/Blockchain/NullPolicy.cs#L23-L24).

Reply:
Oh it's for testing.  Fair enough.